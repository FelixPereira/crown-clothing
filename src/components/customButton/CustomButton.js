import React from 'react';

import './customButton.scss';

const CustomButton = ({ children, ...otherProp }) => {
  return(
    <button className='custom-button' {...otherProp}>
      {children}
    </button>
  )
}

export default CustomButton;