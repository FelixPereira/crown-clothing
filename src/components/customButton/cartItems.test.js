import { shallow } from 'enzyme';
import React from 'react';
import CustomButton from './CustomButton';

it('test custom buttom component', () => {
  expect(shallow(<CustomButton />)).toMatchSnapshot();
})