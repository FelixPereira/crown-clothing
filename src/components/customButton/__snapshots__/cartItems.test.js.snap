// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test custom buttom component 1`] = `ShallowWrapper {}`;
